lisa/wlgen/rta.py

```diff
@@ -408,15 +410,16 @@ def _calibrate(cls, target, res_dir):
 
         logger.info('Target RT-App calibration: %s', pload)
 
-        if 'sched' not in target.modules:
-            return pload
```

Comment:
The return should remain, you'd just have to turn the condition in `if 'cpu-capacities' not in plat_info`